powerauth-push-server/src/main/java/io/getlime/push/service/PushSendingWorker.java

```diff
@@ -250,7 +251,11 @@ private Message buildAndroidMessage(final PushMessageBody pushMessageBody, final
         final AndroidNotification notification = AndroidNotification.builder()
                 .setPriority(deliveryPriority)
                 .setTitle(pushMessageBody.getTitle())
+                .setTitleLocalizationKey(pushMessageBody.getTitleLocKey())
+                .addAllTitleLocalizationArgs(Arrays.asList(pushMessageBody.getTitleLocArgs()))
```

Comment:
We should probably add a null pointer check for `pushMessageBody.getTitleLocArgs()`, otherwise I think that an NPE will be thrown in `Arrays.asList()`.